src/state/shell/progress-guide.tsx

```diff
@@ -167,9 +222,14 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
               subtitle={_(msg`The Discover feed now knows what you like`)}
             />
             <ProgressGuideToast
-              ref={guideCompleteToastRef}
-              title={_(msg`Algorithm training complete!`)}
-              subtitle={_(msg`The Discover feed now knows what you like`)}
+              ref={fifthFollowToastRef}
+              title={_(msg`Half way there!`)}
+              subtitle={_(msg`Follow 10 accounts`)}
+            />
+            <ProgressGuideToast
+              ref={tenthFollowToastRef}
+              title={_(msg`Task complete - follow 10 accounts!`)}
```

Comment:
```suggestion
              title={_(msg`Task complete – follow 10 accounts!`)}
```
maybe with an en-dash?

Reply:
meh, it's already using a wrong dash for a similar experiment, let's clean these up later